cupy/fft/_callback.pyx

```diff
@@ -96,6 +75,38 @@ cdef class _ThreadLocal:
         return tls
 
 
+cdef inline void _set_cupy_paths() except*:
+    # Workaround: older Cython cannot use __file__ in global scope
+    global _cupy_root, _cupy_include, _source_dir
+    if _cupy_root is None:
+        _cupy_root = os.path.join(os.path.dirname(__file__), '..')
+        _cupy_include = os.path.join(_cupy_root, 'core/include')
+        _source_dir = os.path.join(_cupy_root, 'cuda')
+
+
+cdef inline void _set_nvcc_path() except*:
+    # get_nvcc_path() could be a long string like "ccache nvcc ..." from NVCC
+    cdef str nvcc = None
+    cdef str f
+    cdef int i
+
+    global _nvcc
+    if _nvcc == []:
+        nvcc = get_nvcc_path()
+        if nvcc is not None:
+            _nvcc = nvcc.split(' ')
+            # if the host compiler is set in NVCC, we force it to align with
+            # the one reported by sysconfig
+            for i, f in enumerate(_nvcc):
```

Comment:
why don't we use the one that is provided in nvcc?

Reply:
My original thought was I wanna use the compiler Python says it'd use for building extensions, but now think it also makes sense to do the other way around. Will change.

Reply:
Changed in 30a620d. @emcastillo PTAL.